stdlib/internal/debug/debug.flux

```diff
@@ -53,5 +53,8 @@ builtin getOption : (pkg: string, name: string) => A
 //
 builtin feature : (key: string) => A
 
+// Null returns the null value with the given type
+builtin null : (?type: string) => A
```

Comment:
If there were an example here, I suppose it might look like:

```
import "internal/debug"
import "array"

array.from(rows: [{a: 1, b: 2, c: 3}, {a: debug.null("int"), b: 5, c: 6}])
```

Is that the idea? Neat, if so. I will be glad to use this instead of relying on combinations of `union()` and `debug.opaque()`.